src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="personal groups">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dob"
+          >
+        </label>
+
+        <label class="form-field ">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+
+      </fieldset>
```

Comment:
Don't add empty lines between parent and child elements.
```suggestion
        </label>
      </fieldset>
```

Reply:
Fixed